examples/PublishHelloWorldP2P/README.md

```diff
@@ -0,0 +1,28 @@
+# PublishHelloWorld example
+
+This example will show how to send data to the DDS World creating a client publisher.
```

Comment:
DDS World? Is not supposed to use the `ced` middleware?.
Update README with the right setup.

Reply:
Modified!